### DISCUSSION TITLE
TCP with HTTP masquerade behind CDN

### DISCUSSION THREAD

**Discussion by radiumatic** [XTLS/Xray-core#3492] [Q&A] [answered]
Hi,
Does the HTTP Masquerade mimic HTTP standard well enough to be used with a CDN?

**Comment by mmmray** [comment, marked as answer]
No, the only transports that can be used with CDN are:

* websocket
* httpupgrade
* grpc
* splithttp

**Comment by radiumatic** [comment]
splithttp was what I was looking for, thanks.
